src/main/java/com/getindata/connectors/http/internal/table/lookup/HttpLookupTableSourceFactory.java

```diff
@@ -52,21 +55,22 @@ public DynamicTableSource createDynamicTableSource(Context dynamicTableContext)
         FactoryUtil.TableFactoryHelper helper =
             FactoryUtil.createTableFactoryHelper(this, dynamicTableContext);
 
-        ReadableConfig readable = helper.getOptions();
+        ReadableConfig readableConfig = helper.getOptions();
```

Comment:
I think @grzegorz8 changed this name a month ago in #109 form readableConfig to readable.
Im ok with both names but I would like to avoid changing this back and forth :)

Can we agree to one name?
@grzegorz8 @davidradl <- up to you guys

Reply:
reverted